workitem/workitemtype_repository.go

```diff
@@ -111,8 +112,9 @@ func ClearGlobalWorkItemTypeCache() {
 
 // Create creates a new work item in the repository
 // returns BadParameterError, ConversionError or InternalError
-func (r *GormWorkItemTypeRepository) Create(ctx context.Context, spaceID uuid.UUID, id *uuid.UUID, extendedTypeID *uuid.UUID, name string, description *string, icon string, fields map[string]FieldDefinition) (*WorkItemType, error) {
+func (r *GormWorkItemTypeRepository) Create(ctx context.Context, spaceID uuid.UUID, id *uuid.UUID, extendedTypeID *uuid.UUID, name string, description *string, icon string, fields map[string]FieldDefinition, categories []*uuid.UUID) (*WorkItemType, error) {
```

Comment:
@DhritiShikhar never mind my previous comment. I just found that you have created a [function to associate a category with a WIT](https://github.com/fabric8-services/fabric8-wit/commit/33b5fa41ea72643e242d9c72b7442acaeda9a16f#diff-086b748e9f407ea9218c24b8daf660d8R170).